puppet/modules/profiles/manifests/discourse.pp

```diff
@@ -2,8 +2,25 @@
 # @see https://github.com/discourse/discourse/blob/main/docs/INSTALL-cloud.md
 class profiles::discourse {
   $root = '/var/discourse'
-  # TODO: manage docker
-  # TODO: vcsrepo https://github.com/discourse/discourse_docker.git on $root
+
+  if $facts['os']['family'] == 'RedHat' {
+    yumrepo { 'docker-ce-stable':
+      descr   => 'Docker CE Stable - $basearch',
+      baseurl => 'https://download.docker.com/linux/centos/$releasever/$basearch/stable',
+      gpgkey  => 'https://download.docker.com/linux/centos/gpg',
+    }
+
+    ensure_packages(['docker-ce'], { require => Yumrepo['docker-ce-stable'] })
+
+    service { 'docker':
+      ensure  => 'running',
+      enable  => true,
+      require => Package['docker-ce'],
+    }
```

Comment:
should this go to an own module? I kinda didn't want to pull in https://github.com/puppetlabs/puppetlabs-docker, especially as it doesn't even officially support el9 :disappointed:

Reply:
I'm fine with it here, given we only need it once.